docs/specs/xref.yml

```diff
@@ -715,3 +715,27 @@ outputs:
     ["info","at-xref-not-found","Cross reference not found: '<xref href=\"f\" data-raw-source=\"@mentionf\">'","docs/a.md",5,12]
     ["info","at-xref-not-found","Cross reference not found: '<p>\n  <xref href=\"e\" data-raw-source=\"@mentione\" />\n  some text\n  <xref href=\"f\" data-raw-source=\"mentionf\" />\n</p>'","docs/a.md",6,18]
     ["warning","xref-not-found","Cross reference not found: '<p>\n  <xref href=\"e\" data-raw-source=\"@mentione\" />\n  some text\n  <xref href=\"f\" data-raw-source=\"mentionf\" />\n</p>'","docs/a.md",6,78]
+---
+# uid resolving should respect the order of xref settings
+inputs:
+  docfx.yml: |
+    baseUrl: https://docs.microsoft.com
+    xref: 
+      - 1.xrefmap.json
+      - 2.xrefmap.json
+  docs/a.md: Link to @System.String
+  1.xrefmap.json: |
+    {
+      "references":[{
+          "uid": "System.String",
+          "name": "String.1",
+          "fullName": "System.String.1",
```

Comment:
simplify the test case to only include `uid` and `href` ?

Reply:
sounds good